test/reducersTest.js

```diff
@@ -0,0 +1,115 @@
+import trumpMM from '../src/Application/Redux/reducers';
+import StateUtils from '../src/Application/Redux/StateUtils';
+import createAction from '../src/Application/Redux/actions';
+import { COLLECT_INCOME } from '../src/Application/Redux/actions';
+import _ from 'underscore';
+
+import chai from 'chai';
+const assert = chai.assert;
+const expect = chai.expect;
```

Comment:
because I copied and pasted this from Dynamo-table-service and was being dumb, what don't you get about that?